src/Utilities/matrix_of_constraints.jl

```diff
@@ -523,6 +526,19 @@ function load_constants(
     return
 end
 
+load_constants(::Vector, ::Any, ::MOI.AbstractVectorSet) = nothing
```

Comment:
It would probably be safer to be opt-in rather than opt-out here.

Reply:
That's breaking. And it's a bit of a pain to have to enumerate everything. Really, the failing tests in SCS should have picked this up earlier. But I obviously excluded them during development and didn't revisit them :(